TextStyling-Kotlin/app/src/androidTest/java/com/android/example/text/styling/renderer/MarkdownBuilderTest.kt

```diff
@@ -34,9 +34,9 @@ import org.junit.Test
 class MarkdownBuilderTest {
 
     private val context = InstrumentationRegistry.getTargetContext()
-    private val bulletPointColor = context.getColor(R.color.colorAccent)
-    private val codeBackgroundColor = context.getColor(R.color.code_background)
-    private val codeBlockTypeface = ResourcesCompat.getFont(context, R.font.inconsolata)
+    val bulletPointColor = ContextCompat.getColor(context, R.color.colorAccent)
```

Comment:
Use `getColorCompat` from `Extensions.kt`

Reply:
Cool - missed that nice extension.   Good stuff.  Done